Functions with the same name in different namespaces collide

The Xamarin iOS SDK has both a [`WatchKit.WKErrorCode`](https://docs.microsoft.com/en-us/dotnet/api/watchkit.wkerrorcode?view=xamarin-ios-sdk-12) enum and a [`WebKit.WKErrorCode`](https://docs.microsoft.com/en-us/dotnet/api/webkit.wkerrorcode?view=xamarin-ios-sdk-12) enum. But currrently the default naming for the type fetching functions in `bindingmetadata.(iphone|macos).swift` is just `MDW_{typename}`, where `typename` does *not* include the namespace. This prevents `make` from working, with an error like the following:
```
bindingmetadata.iphone.swift:2898:13: error: invalid redeclaration of 'MDW_WKErrorCode()'
public func MDW_WKErrorCode() -> Any.Type
           
bindingmetadata.iphone.swift:2892:13: note: 'MDW_WKErrorCode()' previously declared here
public func MDW_WKErrorCode() -> Any.Type
```
So the function below, in `Program.cs` in type-o-matic, needs to change to fix these collisions:
```
static string FuncIDForTypeDefinition (TypeDefinition type)
{
	// Used to be MetaDataWrapperFor
	// The size of this symbol gets magnified by the number of types.
	// Removing 14 characters saves ~32K
	return $"MDW_{type.Name}";
}
```

--- a/type-o-matic/Program.cs
+++ b/type-o-matic/Program.cs
@@ -176,7 +176,7 @@ namespace typeomatic {
 			// Used to be MetaDataWrapperFor
 	    		// The size of this symbol gets magnified by the number of types.
 			// Removing 14 characters saves ~32K
-			return $"MDW_{type.Name}";
+			return $"MDW_{type.Namespace}_{type.Name}";
 		}
 
 		// for debugging

